File: 240117/더 긴 문자열/longer-string.py
arr = input().split()

word_a = arr[0]
word_b = arr[1]

if len(word_a) > len(word_b):
    print(word_a, len(word_a))
elif len(word_a) < len(word_b):
    print(word_b, len(word_b))
else:
    print("same")

File: 250427/더 긴 문자열/longer-string.js
const fs = require("fs");

const [firstWord, secondWord] = fs.readFileSync(0).toString().trim().split(" ");

function getMoreLengthWord(wordA, wordB) {

    let wordLengthA = wordA.length;
    let wordLengthB = wordB.length;

    if (wordLengthA === wordLengthB) {
        return "same";
    } else if (wordLengthA > wordLengthB) {
        return [wordA, wordLengthA].join(" ");
    } else {
        return [wordB, wordLengthB].join(" ");
    };
};

console.log(getMoreLengthWord(firstWord, secondWord));